cypress/integration/Crosscutting/PageOptionsTest.feature

```diff
@@ -0,0 +1,104 @@
+            | /types/breast/research                                                                      | mobile     | font resizer,Print |
+            | /espanol/tipos/seno/investigacion                                                           | tablet     | font resizer,Print |
+            | /about-cancer/coping/feelings/relaxation/dfharvard                                          | mobile     | font resizer,Print |
+            | /about-cancer/coping/feelings/relaxation/2019-investigators-site                            | tablet     | font resizer,Print |
+            | /news-events/press-releases/2019/annual-report-nation-2018                                  | mobile     | font resizer,Print |
+            | /espanol/noticias/comunicados-de-prensa/2019/informe-anual-a-la-nacion-1975-2014            | tablet     | font resizer,Print |
+            | /about-cancer/treatment/side-effects/fatigue/fatigue-pdq                                    | mobile     | font resizer,Print |
+            | /espanol/cancer/tratamiento/efectos-secundarios/fatiga/fatiga-pdq                           | tablet     | font resizer,Print |
+            | /news-events/cancer-currents-blog                                                           | mobile     | none,Print         |
+            | /espanol/noticias/temas-y-relatos-blog                                                      | tablet     | none,Print         |
+            | /types/breast                                                                               | mobile     | none,Print         |
+            | /espanol/tipos/seno                                                                         | tablet     | none,Print         |
+            | /about-cancer                                                                               | mobile     | none,Print         |
+            | /espanol/cancer                                                                             | tablet     | none,Print         |
+            | /news-events/press-releases/2019                                                            | mobile     | none,Print         |
+            | /espanol/noticias/comunicados-de-prensa/2019                                                | tablet     | none,Print         |
+            | /about-nci/organization/nci-at-a-glance                                                     | mobile     | none,Print         |
+            | /espanol/infografia-nci                                                                     | tablet     | none,Print         |
+            | /about-cancer/coping/feelings/relaxation/hedge-maze                                         | mobile     | none,Print         |
+
+    Scenario Outline: Verify that font resizer button exists on the pages
+        Given user is navigating to "<url>"
+        Then page options are displayed
+        And "<pageOption2>" button is displayed
```

Comment:
here you can just pass font resizer directly, since it is the only option being tested here - and get rid of the column pageOption2

 And "font resizer" button is displayed